.travis.yml

```diff
@@ -1,22 +1,41 @@
 language: node_js
 env:
-  - ASCIIDOCTOR_CORE_VERSION=v1.5.5
-  - ASCIIDOCTOR_CORE_VERSION=master
+- ASCIIDOCTOR_CORE_VERSION=v1.5.5
+- ASCIIDOCTOR_CORE_VERSION=master
 node_js:
-  - "4"
-  - "6"
-  - "node"
-
+- '4'
+- '6'
+- node
 jdk:
-  - oraclejdk8
-
-before_script:
-  - npm install
-
+- oraclejdk8
 script:
- - npm run lint
- - npm run package
- - npm run docs
- - SKIP_BUILD=1 npm run examples
-
+- npm run lint
+- npm run package
+- npm run docs
+- SKIP_BUILD=1 npm run examples
 sudo: false
+
+before_deploy:
+- npm run dist
+- if [[ "$TRAVIS_BRANCH" == master ]]; then npm --no-git-tag-version version "$(npm view . version)+git.$(git log -n1 --pretty=format:%h)"; fi
+- if [[ "$TRAVIS_BRANCH" == master ]]; then npm config set tag dev; fi
+
+deploy:
+- provider: npm
+  skip_cleanup: true
+  email: $NPM_USER_EMAIL
+  api_key: $NPM_USER_API_KEY
+  on:
+    tags: true
+    repo: asciidoctor/asciidoctor.js
+    condition: "$ASCIIDOCTOR_CORE_VERSION = 'v1.5.5'"
```

Comment:
If we don't want to publish on `master` then we can use the condition `"$ASCIIDOCTOR_CORE_VERSION = 'v1.5.5'"`.

However `v1.5.5` is "static" so every time a new version comes out we need to change it. In order to avoid that, could we use the condition "$ASCIIDOCTOR_CORE_VERSION -ne 'master'" or `"$ASCIIDOCTOR_CORE_VERSION =~ ^v.*"` ?
As far as I understand we can use Bash condition ? https://docs.travis-ci.com/user/deployment/#Conditional-Releases-with-on%3A)